Django-Start-Project.md

```diff
@@ -0,0 +1,265 @@
+# Creating a Project with Django
+
+[Home](Python)
+
+Now that we know how to create virtual environments and use pip, we can begin
+building our project. In this article, we will start our first Django project,
+create tests, and start our development server.
```

Comment:
> In this article, we will start our first Django project, create tests, and start our development server.

The word _start_ feature twice in this sentence. Maybe make it somewhat like this?

> In this article, we will create our first Django project, write unit tests, and start our development server.